Makefile.common

```diff
@@ -36,6 +36,9 @@ endif
 endif
 
 ifeq ($(STATIC_LINKING), 1)
+ifeq ($(platform), ctr)
+SOURCES_C +=   $(LIBRETRO_COMM_DIR)/audio/audio_mixer.c
+endif
 else
```

Comment:
this is probably already added there, isn't it?
https://github.com/libretro/mrboom-libretro/blob/1c8b5fc70ac3c2e2cc369022ab5d517a493a7c16/Makefile.common#L62

Reply:
No, STATIC_LINKING is 1, so line 62 is never executed